content-src/experiments/tracking-protection.yaml

```diff
@@ -91,6 +91,113 @@ created: '2016-09-22T00:07:28.847430Z'
+   contributes to website breakage. The experiment was fairly 
+   simple by design: we enabled Tracking Protection and let users disable it on specific sites 
+   or provide reports about websites that seemed to break as a result of the experiment.</p>
+  <p><strong>Here’s how it went down</strong></p>
+  <p>The user interface for the Tracking Protection experiment was simple. When Tracking 
+  Protection was enabled, users saw an icon in their browser’s URL bar. Clicking 
+  the icon opened a panel where participants could report a problem and turn 
+  Tracking Protection off for that domain.</p>
+  <figure>
+  <img src="/static/images/experiments/tracking-protection/experiments_experimentdetail/detail1.jpg" alt="Tracking Protection doorhanger" />
+  </figure>
+  <p>Participants had the option to give more details about the problems they saw.</p>
+  <figure>
+  <img src="/static/images/experiments/tracking-protection/experiments_experimentdetail/detail2.jpg" alt="Tracking Protection doorhanger" />
+  </figure>
+  <p>For this experiment, we only collected data when participants turned Tracking 
+  Protection on or off, or when they reported problems with a particular domain. 
+  We did not collect data on other browsing activity and we did not record full URLs.</p>
+  <p><strong>Here’s what we learned</strong></p>
+  <p>We hypothesized that users would frequently disable Tracking Protection on broken sites. 
+  This, it turns out, was not the case. The chart below shows that despite a high 
+  number of participants using Firefox Tracking Protection every day, Tracking 
+  Protection was disabled on very few sites.</p>
+  <p><iframe src="https://docs.google.com/spreadsheets/d/19sMMr2Z6O8Ny_Olisz2sKE13CLIOet9KAlrcn4HSXDk/pubchart?oid=1101471247&amp;format=interactive" width="600" height="380" frameborder="0"></iframe></p>
+  <figure>
+  <figcaption>The number of times users disabled Tracking Protection was low to begin with,  and decreased over the life of the experiment.</figcaption>
+  </figure>
+  <p>Further, the rate at which users disabled Tracking Protection decreased as the 
+  experiment progressed. We think that once users disabled Tracking Protection for 
+  the most problematic sites, they didn’t encounter or weren’t bothered by breakage 
+  to the point that they would disable it for additional sites they visited later in 
+  the experiment.  Since we didn’t measure total page views, there are limits to what 
+  we can conclude about the number of sites broken by Tracking protection. Nevertheless, 
+  this data suggests that breakage may be less widespread than we initially believed, 
+  and it’s worth digging deeper into more specific breakage data.</p>
+  <p><iframe src="https://docs.google.com/spreadsheets/d/19sMMr2Z6O8Ny_Olisz2sKE13CLIOet9KAlrcn4HSXDk/pubchart?oid=206050478&amp;format=interactive" width="600" height="380" frameborder="0"></iframe></p>
```

Comment:
ultra nit: **eslint-plugin-jsx-a11y** says:

```sh
➜  jsx-a11y-test $ npm test

> jsx-a11y-test@1.0.0 test /Users/pdehaan/dev/tmp/del/jsx-a11y-test
> eslint .

/Users/pdehaan/dev/tmp/del/jsx-a11y-test/index.js
  35:6  error  <iframe> elements must have a unique title property  jsx-a11y/iframe-has-title
  47:6  error  <iframe> elements must have a unique title property  jsx-a11y/iframe-has-title
  79:6  error  <iframe> elements must have a unique title property  jsx-a11y/iframe-has-title

✖ 3 problems (3 errors, 0 warnings)

npm ERR! Test failed.  See above for more details.
```

(Using a hacked empty dir and the following .eslintrc.yml config:)

```yaml
env:
  es6: true
  node: true

extends:
  - eslint:recommended
  - plugin:jsx-a11y/recommended

plugins:
  - jsx-a11y

root: true
```